.changeset/little-ads-return.md

```diff
@@ -0,0 +1,33 @@
+---
+"@makeswift/runtime": minor
+---
+
+BREAKING: The latest upgrade to the Makeswift client `getPages` method
+introduces sorting, path filtering, and pagination. This method was not
+previously paginated - in order to get all your pages, you must now paginate
+through results:
+
+```tsx
+import { client } from '@/makeswift/client'
+import { MakeswiftPageResult } from '@makeswift/runtime/next'
+
+async function getAllPages(): Promise<MakeswiftPageResult[]> {
+  let hasMorePages = true
+  let after: string | undefined = undefined
+  const pages: MakeswiftPageResult[] = []
+
+   do {
+    const { data, hasMore } = await client.getPages({ after })
+    pages.push(...data)
+    hasMorePages = hasMore
+    after = data.at(-1)?.id
+  } while (hasMorePages)
+
+  return pages
+}
+
+This change also deprecates the client `getSitemap` method, with the
+recommendation that sitemaps should now be generated using data returned from
+`getPages`.
```

Comment:
A link to the upgrade guide or even a quick example inline showing how to migrate would be super helpful!